<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit</class>
 <widget class="QWidget" name="Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="tableLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="addTableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Cancel and Close</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirmButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Confirm your selections</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
